pyproject.toml

```
@@ -4,7 +4,7 @@ version = "0.0.0"
 description = "Introductions to key concepts in quantum machine learning, as well as tutorials and implementations from cutting-edge QML research."
 readme = "README.md"
 license = "Apache-2.0"
-requires-python = ">=3.10.0"
+requires-python = ">=3.10.0,<3.13.0"
```

While our CI (for pennylane + plugins) only tests and guarantees support for Python 3.11-3.13, we shouldn't block users from running our code on newer versions like 3.14, which are likely to be compatible.

```suggestion
requires-python = ">=3.10.0"
```

Also ... remind me again why this can't be bumped to 3.11 now that we have eliminated the packages that conflict with this version?

---

```
The current project's supported Python range (>=3.10.0) is not compatible with some of the required packages Python requirement:
  - mitiq requires Python <3.13,>=3.10, so it will not be installable for Python >=3.13

Because qml depends on mitiq (0.43.0) which requires Python <3.13,>=3.10, version solving failed.

  * Check your dependencies Python requirement: The Python requirement can be specified via the `python` or `markers` properties

    For mitiq, a possible solution would be to set the `python` property to ">=3.10.0,<3.13"

```